db/migrate/20140328041648_create_api_applications.rb

```diff
@@ -1,15 +0,0 @@
-class CreateApiApplications < ActiveRecord::Migration[5.1]
```

Comment:
Let's leave this migration in place. We should instead generate a new migration that removes this column. That way we keep database schema history in tact.

```rails g migration remove_api_applications```

```
def change
  drop_table :api_applications
end
```

Should do it

Reply:
added new commit with these changes